components/scream/src/physics/p3/p3_ice_collection_impl.hpp

```diff
@@ -21,9 +21,9 @@ ::ice_cldliq_collection(
   constexpr Scalar tmelt  = C::Tmelt;
 
   // set up masks
-  const auto t_is_negative        = temp < tmelt;
-  const auto qi_incld_gt_small = qi_incld > qsmall;
-  const auto qc_incld_gt_small    = qc_incld > qsmall;
+  const auto t_is_negative        = temp <= tmelt;
```

Comment:
If you look at the fortran, it's doing `.ge.` and `.le.` , so this port was not exactly true to the reference. I'm not sure which implementation is better, but I went with the fortran.

Reply:
Yes, we should follow the Fortran or otherwise correct it. In this case, let's follow it.

Reply:
One tiny cosmetic thing is you could change `_gt_` on the LHS to `_ge_`, following the C++ naming convention elsewhere, e.g., in the routine following this one.